# core/exceptions.py
class AthentoseError(Exception):
    pass